landing-pages/site/content/en/survey/_index.html

```diff
@@ -1,26 +1,28 @@
 ---
-title: "Airflow Survey 2023"
-linkTitle: "Airflow Survey 2023"
+title: "Airflow Survey 2024"
+linkTitle: "Airflow Survey 2024"
 menu:
   main:
     weight: 31
 ---
 
 <div>
-  <img src="Astronomer - Airflow Survey 2023 - Results - Demographics - Landscape@2x.png"/>
-  <img src="Astronomer - Airflow Survey 2023 - Results - Community and Contribution - Landscape@2x.png"/>
-  <img src="Airflow Survey 2023 Results - Airflow Usage - Page 1 - Revised - Landscape@2x.png"/>
-  <img src="Astronomer - Airflow Survey 2023 - Results - Airflow Usage - Page 2 - Landscape@2x.png"/>
-  <img src="Airflow Survey 2023 Results - Airflow Usage - Page 3 - Revised - Landscape@2x.png"/>
-  <img class="bottom-image" src="Astronomer - Airflow Survey 2023 - Results - Future - Landscape@2x.png"/>
+  <img src="2024/Airflow Survey 2024 Results.png" alt="Airflow Survey 2024 Infographic" />
 </div>
 
 <div class="survey-button-container">
   <div class="survey-button">
-    <a href="https://docs.google.com/forms/d/1wYm6c5Gn379zkg7zD7vcWB-1fCjnOocT0oZm-tjft_Q/viewanalytics">
+    <a href="https://astronomer.typeform.com/report/SF2VGNTc/fRSeRcKKJ3kgYXVl">
       <button type="submit" class="btn-hollow btn-brown btn-with-icon with-box-shadow">
         <span class="bodytext__medium--brownish-grey">View Raw Data</span>
       </button>
     </a>
   </div>
 </div>
+
+<div class="past-surveys">
+  <h3>Past Surveys:</h3>
```

Comment:
Thinking about it a little more, I think we should move these back to the blog. The blog is already where [2019](https://airflow.apache.org/blog/airflow-survey/), [2020](https://airflow.apache.org/blog/airflow-survey-2020/), and [2022](https://airflow.apache.org/blog/airflow-survey-2022/) are and it gives us the benefit of the ["survey" tag](https://airflow.apache.org/blog/tags/survey/). We can keep or get rid of this "past surveys" list then as well (or, it can stay, but it should at least have all of them 😄).

Reply:
Changed it.